### PR TITLE
Doc fixes

### LINKED ISSUE #493: documentation mention `throwErr` instead of `throwError`a
https://hackage.haskell.org/package/servant-server-0.7/docs/Servant-Server-Internal-ServantErr.html
https://hackage.haskell.org/package/servant-server-0.7/docs/doc-index-T.html

### PR DESCRIPTION
**Pull request by jkarni**
Fixes #493 and #498

### CODE REVIEW

**Comment by jkarni** [comment]
Also removes link to 0.4 version of tutorial, since stackage has 0.5 and the 0.4 version is broken in various places and should be deprecated. (See also #483 )

File: doc/tutorial/index.rst [hunk 1]
```diff
@@ -3,12 +3,7 @@ Tutorial
...
 This is an introductory tutorial to **servant**.
 
-.. note::
-   This tutorial is for the latest version of servant.  The tutorial for
-   servant-0.4 can be viewed
-   `here <https://haskell-servant.github.io/tutorial/>`_.
-
-(Any comments, issues or feedback about the tutorial can be handled
+(Any comments, issues or feedback about the tutorial can be submitted
 through
```
Comment (soenkehahn) [inline doc/tutorial/index.rst]: `submitted to`?

**Comment by soenkehahn** [comment]
Apart from my comments, LGTM.

**Comment by alpmestan** [comment]
👍

**Pull request merged**